src/pass.ts

```diff
@@ -287,13 +287,14 @@ export class Pass {
 
 	beacons(resetFlag: null): this;
 	beacons(...data: schema.Beacon[]): this 
-	beacons(...args: (schema.Beacon|null)[]){
-		if (args[0] === null) {
+	beacons(...data: (schema.Beacon|null)[]){
+
```

Comment:
Just to make it perfect, what about this blank line? 😂

Reply:
removed